test/Date.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import expect from 'expect';
+import { shallow } from 'enzyme';
+import Date from '../js/components/Message/Types/JSObject/Date';
+import data from "./data/Date.js";
+
+describe('Date', () => {
+
+  data.forEach((testObj, testName) => {
+    it(`should render a ${testName}`, () => {
+      const message = testObj.responsePacket.result;
+
+      const rendered = shallow(<Date message={message} />);
+      expect(rendered.text()).toContain(testObj.consoleOutput);
```

Comment:
Agreed, I had been thinking the same thing. 

Do you think we'd want to restructure the HTML as part of the rewrite? If we do that, it might be good to keep it as just a text check while that's going on. But I think we'll want to check against the HTML structure in the final tests.